scripts/fuzz_shell.js

```diff
@@ -166,10 +165,6 @@ var imports = {
     'log-f32': logValue,
     'log-f64': logValue,
   },
-  'env': {
-    'setTempRet0': function(x) { tempRet0 = x },
-    'getTempRet0': function() { return tempRet0 },
-  },
```

Comment:
Don't we need to keep providing the imports for now, to avoid fuzzer breakage?

If we don't, then who is responsible for creating and providing those functions after this PR>